source/Classes/020.SChannelProtocolServer.ps1

```
@@ -0,0 +1,25 @@
+[DscResource()]
```

PSScriptAnalyzer (1.24.0)

[error] The DSC resource 'SChannelProtocolServer' must have at least one key property (using the syntax [DscProperty(Key)].)

(DscResourceMissingKeyProperty)